dev/com.ibm.ws.org.apache.myfaces.2.3/src/com/ibm/ws/jsf/taglibconfig/JSFGlobalTagLibConfig.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
```

Comment:
Hey we should change the copyright date don't you think since we're changing the package these reside in?

Reply:
In mind I saw this as the same package, but, you're right, I should update the copyrights as the name had changed.

Reply:
Should be good now